### PR TITLE
Fix Node Health not live & Add tooltip

### PR DESCRIPTION
**Pull request by RMartinOscar**
- [x] Add livewire poll so it auto refresh
- [x] Add back version tooltip

image
image

### CODE REVIEW

**Comment by Boy132** [comment]
> Nothing is shown on hover if the node is offline should i add a `Unknown` ?

Yes, something similar to the tooltip from the old admin area.

We should also [lower the refresh rate](https://livewire.laravel.com/docs/wire-poll#timing-control) to 5 seconds (or maybe 10 or 15 even?)

**Comment by RMartinOscar** [comment]
image
image

Maybe its a bit long ?

I searched for a way to define [$pollingInterval](https://filamentphp.com/docs/3.x/widgets/stats-overview#live-updating-stats-polling) in php but it didn't care so i hardcoded it in the blade

**Comment by Boy132** [comment]
> image image
> 
> Maybe its a bit long ?

Is there a way to add a line break?

**Comment by RMartinOscar** [comment]
> Is there a way to add a line break?

I tried without success

**Comment by Boy132** [comment]
On a second thought, do we even print errors to the browser console?

**Comment by RMartinOscar** [comment]
> On a second thought, do we even print errors to the browser console?

LMAO we don't

File: app/Models/Node.php [hunk 1]
```diff
@@ -307,9 +307,7 @@ public function systemInformation(): array
 
                 if ($message->startsWith('cURL error 6: Could not resolve host')) {
                     $message = str('Could not resolve host');
-                }
-
-                if ($message->startsWith('cURL error 28: Failed to connect to ')) {
+                } elseif ($message->startsWith('cURL error 28: Failed to connect to ')) {
```
Comment (lancepioch) [inline app/Models/Node.php:310]: You're triggering me, this was fine before

```suggestion
                }
                
                if ($message->startsWith('cURL error 28: Failed to connect to ')) {
```

File: app/Models/Node.php [hunk 2]
```diff
@@ -308,7 +308,7 @@ public function systemInformation(): array
                 if ($message->startsWith('cURL error 6: Could not resolve host')) {
                     $message = str('Could not resolve host');
                 }
-
+
```
Comment (lancepioch) [inline app/Models/Node.php:311]:
```suggestion

```

File: app/Models/Node.php [hunk 3]
```diff
@@ -308,7 +308,6 @@ public function systemInformation(): array
                 if ($message->startsWith('cURL error 6: Could not resolve host')) {
                     $message = str('Could not resolve host');
                 }
```
Comment (lancepioch) [inline app/Models/Node.php:310]:
```suggestion
                }

```

**Review by lancepioch — APPROVED** [review]

**Pull request merged**